runtime/runtime_test.go

```diff
@@ -10634,123 +10634,6 @@ func TestRuntimeNonPublicAccessModifierInInterface(t *testing.T) {
 	require.Len(t, conformanceErr.MemberMismatches, 2)
 }
 
-func TestRuntimeMoveSelfVariable(t *testing.T) {
```

Comment:
These fails at the checker now. There are checker tests added for this particular scenario.